content/docs/user-guide/components/reference/terrain/surface-material-list.md

```diff
@@ -32,14 +48,21 @@ Use the following request functions with the `TerrainAreaMaterialRequestBus` EBu
 
 | Request Name | Description | Parameter | Return | Scriptable |
 |-|-|-|-|-|
+| `GetTerrainSurfaceMaterialRegion` | Retrieves the Aabb for the region where a `TerrainSurfaceMaterialMapping` exists.  | None | Aabb | No. |
 | `GetSurfaceMaterialMappings` | Retrieves all the assigned surface types, the materials that you have assigned to them, and the bounds that are set for this entity.  | None | Terrain Surface Material Mapping: Vector | No. |
```

Comment:
Very nit - It would be better if you can remove the periods here and the next comment
```suggestion
| `GetTerrainSurfaceMaterialRegion` | Retrieves the Aabb for the region where a `TerrainSurfaceMaterialMapping` exists.  | None | Aabb | No |
| `GetSurfaceMaterialMappings` | Retrieves all the assigned surface types, the materials that you have assigned to them, and the bounds that are set for this entity.  | None | Terrain Surface Material Mapping: Vector | No |
```

Reply:
Will do